jenkins-integration/jenkins-jobs/scenarios/multi-pass-scenario/Jenkinsfile

```diff
@@ -4,12 +4,20 @@ node {
 }
 pipeline.multipass_pipeline([
         [job_name: 'firsttest',
+         server_version: [
+                 type: "pe",
+                 pe_version: "2016.2.0"
```

Comment:
This seems fine to me for now.  Once we have support for modifying settings before a run (HOCON, etc.), we could make one of these have 'environment_timeout' set to on and one set to off, or something along those lines.